blocked-edges/4.14.40-OVNlibreswan.yaml

```diff
@@ -7,6 +7,14 @@ matchingRules:
 - type: PromQL
   promql:
     promql: |
-      group by (resource) (max_over_time(apiserver_storage_objects{_id="",resource="egressips.k8s.ovn.org"}[1h]))
-      or on ()
+      group by (ipsec) (label_replace(max_over_time(ovnkube_controller_ipsec_enabled{_id=""}[1h]), "ipsec", "enabled (4.14)", "", "") == 1)
```

Comment:
the `ovnkube_controller_ipsec_enabled` gauge metric is at zone/node level in 4.14, so this is set from every node. are we ok with this ?

Reply:
`group by (ipsec) (...)` will collapse matching `ovnkube_controller_ipsec_enabled` per `ipsec` label, and the `label_replace` ensures only a single `enabled (4.14)` value for that label.  So that means that a single node reporting a value of 1 for that metric will get the cluster treated as IPsec-enabled as far as this risk is concerned.  Which sounds like the semantics we want.

Reply:
ok @wking , that's great.